app/views/shared/check_answers/_laspo.html.erb

```diff
@@ -1,14 +1,16 @@
 <section>
-  <h2 class="govuk-heading-m govuk-!-padding-top-6"><%= t(".heading") %></h2>
-  <%= govuk_summary_list(card: { title: t(".heading") }, actions: !read_only) do |summary_list| %>
-    <%= summary_list.with_row(html_attributes: { id: "app-check-your-answers__in_scope_of_laspo" }) do |row| %>
-      <%= row.with_key(text: t(".question"), classes: "govuk-!-width-one-half") %>
-      <%= row.with_value(text: yes_no(@legal_aid_application.in_scope_of_laspo)) %>
-      <%= row.with_action(
-            text: t("generic.change"),
-            href: providers_legal_aid_application_in_scope_of_laspo_path(@legal_aid_application),
-            visually_hidden_text: t(".question"),
-          ) %>
+  <%= govuk_summary_card(title: t(".heading"), html_attributes: { id: "app-check-your-answers__laspo" }) do |card| %>
+    <% unless read_only %>
+      <% card.with_action do %>
+        <%= govuk_link_to(t("generic.change"),
+                          providers_legal_aid_application_in_scope_of_laspo_path(@legal_aid_application)) %>
+      <% end %>
+    <% end %>
+    <%= card.with_summary_list(actions: !read_only) do |summary_list| %>
```

Comment:
This action can also be false I think, for the same reason as the other comment

Reply:
👍 removed (actions: !read_only) as I don't think this is required